scripts/fuzz_opt.py

```diff
@@ -178,7 +185,12 @@ def pick_initial_contents():
     # half the time don't use any initial contents
     if random.random() < 0.5:
         return
-    test_name = random.choice(all_tests)
+    # some of the time use initial contents that are known to be especially
+    # important
+    if random.random() < 0.5:
+        test_name = random.choice(IMPORTANT_INITIAL_CONTENTS)
```

Comment:
Is that single pass worth half initial contents testing? (I'm not very familiar with things there myself, just wondering..)

Reply:
Maybe half is a little high, yeah, I'm not sure. But it is the pass that is getting a lot of new commits, every day basically - and we've found bugs. The other initial contents have not found bugs for weeks now.